src/includes/class-alma-wc-settings.php

```diff
@@ -255,13 +261,13 @@ public function get_enabled_plans_definitions() {
 	private function get_i18n( $key ) {
 
 		if ( Alma_WC_Internationalization::is_site_multilingual() ) {
-			if ( $this->{$key . '_' . get_user_locale() } ) {
-				return $this->{$key . '_' . get_user_locale() };
+			if ( $this->{$key . '_' . get_locale() } ) {
```

Comment:
are you sure that will works fine ?
try with site general settings in english and administrator in spanish please ?

Reply:
That works.

Reply:
That works.
get_user_locale() returns the favorite user language in back-office, and our need is to get the current site language (selected with WPML language switcher), which is returned by get_locale().